src/DurableSDK/PowerShellServices.cs

```diff
@@ -6,50 +6,144 @@
 namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
 {
     using System;
+    using System.Collections.ObjectModel;
+    using System.Linq;
     using System.Management.Automation;
-    using PowerShell;
+    using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+    using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
+    using Newtonsoft.Json;
 
     internal class PowerShellServices : IPowerShellServices
     {
-        private const string SetFunctionInvocationContextCommand =
-            "Microsoft.Azure.Functions.PowerShellWorker\\Set-FunctionInvocationContext";
+        private readonly string SetFunctionInvocationContextCommand;
+        private const string ExternalDurableSDKName = "DurableSDK";
```

Comment:
note to self: before merging, we need a proper namespace reserved for the external SDK.